Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -1163,7 +1163,7 @@ static Color normalizedColor(Color color, bool ignoreDefaultColor, Element& elem
 void HTMLConverter::_newLineForElement(Element& element)
 {
     unichar c = NSLineSeparatorCharacter;
-    RetainPtr<NSString> string = adoptNS([[NSString alloc] initWithCharacters:&c length:1]);
+    RetainPtr<NSString> string = adoptNS([[NSString alloc] initWithCharactersNoCopy:&c length:1 freeWhenDone:NO]);
```

Comment:
Is it documented that `-[NSString replaceCharactersInRange:withString:]` is guaranteed to never retain the passed in string, even in unusual cases like a one-character string? Or are we just guessing based on common sense?

Reply:
I actually heard that these one letter strings are handled differently in practice but that's not at all why I did this: I think that copying a one line string is not needed.

Reply:
You mean one character string. I think that while possibly appealing, this is a bad idea for a change that will likely actually make performance a tiny bit worse.

Reply:
I would like to see this addressed before landing.